test/e2e/fixture/applicationsets/utils/fixture.go

```diff
@@ -112,9 +122,15 @@ func EnsureCleanState(t *testing.T) {
 	}
 
 	// Delete the argocd-e2e-external namespace, if it exists
-	err2 := fixtureClient.KubeClientset.CoreV1().Namespaces().Delete(context.Background(), ArgoCDExternalNamespace, v1.DeleteOptions{PropagationPolicy: &policy})
+	err2 := fixtureClient.KubeClientset.CoreV1().Namespaces().Delete(context.Background(), string(ArgoCDExternalNamespace), v1.DeleteOptions{PropagationPolicy: &policy})
 	if err2 != nil && !strings.Contains(err2.Error(), "not found") { // 'not found' error is expected
-		CheckError(err)
+		CheckError(err2)
+	}
+
+	// Delete the argocd-e2e-external namespace, if it exists
+	err3 := fixtureClient.KubeClientset.CoreV1().Namespaces().Delete(context.Background(), string(ArgoCDExternalNamespace2), v1.DeleteOptions{PropagationPolicy: &policy})
+	if err3 != nil && !strings.Contains(err3.Error(), "not found") { // 'not found' error is expected
+		CheckError(err3)
```

Comment:
we should cleanup `argocd-e2e-external2` as well to ensure clean state.

Reply:
@ishitasequeira I think that's what's happening on line 131, right?

Reply:
Ahh my bad. Got confused with the comment added.